client/signup/steps/user/index.jsx

```diff
@@ -149,6 +150,7 @@ export class UserStep extends Component {
 				submitForm={ this.submitForm }
 				submitButtonText={ this.submitButtonText() }
 				suggestedUsername={ this.props.suggestedUsername }
+				isSocialSignupEnabled={ isEnabled( 'signup/social' ) && this.props.flowName === 'social' }
```

Comment:
We could do that, I wanted to avoid making the PR larger though.
Question: is this `social` flow going to be used in production or is this just for us?